tests/std/tests/GH_004930_char_traits_user_specialization/test.cpp

```diff
@@ -280,6 +285,16 @@ CONSTEXPR20 bool test_gh_4930() {
 static_assert(test_gh_4930());
 #endif // _HAS_CXX20
 
+void test_gh_4956() {
+    basic_string<char, odd_char_traits<char>> s("QQPPQ", 5);

```

Comment:
It's nice to have a short description of the issue the regression test is testing for:
```suggestion
    // bitset's stream extraction operator was using `!=` to compare characters instead of `traits::eq`
    basic_string<char, odd_char_traits<char>> s("QQPPQ", 5);
```

Reply:
I also captured both bug titles exactly.